src/Modal/Modal.scss

```diff
@@ -109,3 +109,43 @@
 .modal-footer {
   flex: 0 0 auto;
 }
+
+.pgn__modal-popup__tooltip {
+  margin: .5rem 0;
+  border: $border-width solid rgb(0 0 0 / 20%);
```

Comment:
Why you don't use `rgba(color, alpha)` instead of `rgb(color / alpha)`?

Reply:
IDE says what to do)
Corrected, thanks